x-pack/plugins/maps/public/classes/layers/ems_vector_tile_layer/ems_vector_tile_layer.tsx

```diff
@@ -391,6 +403,23 @@ export class EmsVectorTileLayer extends AbstractLayer {
     });
   }
 
+  _setLanguage(mbMap: MbMap, mbLayer: LayerSpecification, mbLayerId: string) {
+    const locale = this.getLocale();
+    let textProperty;
+    if (locale === null || locale === DEFAULT_EMS_LOCALE) {
+      if (mbLayer.type === 'symbol') {
```

Comment:
How about just returning for the case `if (locale === null || locale === DEFAULT_EMS_LOCALE)`? Line 411 and then 419 are kind of weird in this case, since you are just setting 'text-field' to its current value.